CHANGELOG.md

```diff
@@ -1,9 +1,7 @@
-# Changelog
+# Change Log
```

Comment:
Changelog is one word

```suggestion
# Changelog
```

Reply:
This is what lerna generated with conventional commits.

Reply:
Okay, interesting. Can we configure Lerna to use the proper spelling?